main/CG-template.md

```diff
@@ -0,0 +1,26 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the {FULLDATE} video call of WebAssembly's Community Group
+
+- **Where**: Virtual meeting
+- **When**: {FULLDATE}, 5pm-6pm UTC ({FULLDATE}, 9am-10am Pacific Standard Time)
+- **Location**: *link on W3C [calendar](https://www.w3.org/groups/cg/webassembly/calendar/) or Google Calendar invitation*
+
+### Registration
+
+None required if you've attended before. Send an email to the acting [WebAssembly CG chair](mailto:webassembly-cg-chair@chromium.org)
+to sign up if it's your first time. The meeting is open to CG members only.
+
+## Agenda items
+
+1. Opening
+1. Proposals and discussions
+1. Closure
+
+## Agenda items for future meetings
+
+*None*
+
+## Meeting Notes
+
+To be added after the meeting.
```

Comment:
Optional: Do we want to add an indicator for ending the meeting? It seems superflous, but helpful for shared note taking.

Reply:
Also, closure is on the agenda items, so we can include it here or remove from there.

Reply:
We usually have subheadings for attendees, proposals and discussions, so I added those too, along with closure.